feast_hive/hive.py

```diff
@@ -79,14 +90,22 @@ class HiveOfflineStoreConfig(FeastConfigBaseModel):
     kerberos_service_name: StrictStr = "impala"
     """ Specify particular impalad service principal. """
 
+    use_unique_field_names: StrictBool = False
```

Comment:
instead introduce a new variable here, why not just use `hive_conf` config?
`
hive_conf:
    hive.resultset.use.unique.column.names: false
`